packages/Microsoft.Bot.Components.Telephony/Actions/CallTransferDialog.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AdaptiveExpressions.Properties;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs;
+using Newtonsoft.Json;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Components.Telephony.Actions
+{
+    public class CallTransferDialog : Dialog
+    {
+        [JsonConstructor]
+        public CallTransferDialog([CallerFilePath] string sourceFilePath = "", [CallerLineNumber] int sourceLineNumber = 0)
+            : base()
+        {
+            // enable instances of this command as debug break point
+            this.RegisterSourceLocation(sourceFilePath, sourceLineNumber);
+        }
+
+        [JsonProperty("$kind")]
+        public const string Kind = "CallTransferDialog";
```

Comment:
Proposed name: 

```suggestion
        public const string Kind = "Microsoft.Telephony.CallTransfer";
```

Reply:
I had changed my $kind from CallTransferDialog to Microsoft.Bot.Components.CallTransferDialog and that was causing the issue --  I hear the message that my dialog is not registered with factory. With CallTransferDialog it works

Reply:
Yes, because you also need to update your schema file name and the CallTransferDialog.Kind property to the same thing.

Reply:
Lets get the right name, and then we can help you make it work. But it must start with Microsoft.*